src/teacher.py

```diff
@@ -192,3 +192,33 @@ def get_responses(self) -> List[Optional[Response]]:
     def get_query_cost(self) -> float:
         """Return the query cost."""
         return self._query_cost
+
+
+@dataclass
+class TeacherInteractionMonitorWithVideo(TeacherInteractionMonitor,
+                                         utils.VideoMonitor):
+    """A monitor that wraps interaction with an environment to include
```

Comment:
This docstring is out of date, it can now read:
"""A monitor that wraps a TeacherInteractionMonitor to optionally also render every state and action encountered, if CFG.make_interaction_videos is True."""